components/ProfileMenu.tsx

```diff
@@ -0,0 +1,197 @@
+                <StyledMenuEntry href={`/${LoggedInUser.collective.slug}`}>
+                  <FormattedMessage id="menu.profile" defaultMessage="Profile" />
+                </StyledMenuEntry>
+                <StyledMenuEntry href={`/dashboard/${LoggedInUser.collective.slug}`}>
+                  <FormattedMessage id="Dashboard" defaultMessage="Dashboard" />
+                </StyledMenuEntry>
+                <StyledMenuEntry href={`/dashboard/${LoggedInUser.collective.slug}/info`}>
+                  <FormattedMessage id="Settings" defaultMessage="Settings" />
+                </StyledMenuEntry>
+                {hasAvailablePreviewFeatures && (
+                  <StyledMenuEntry as="button" onClick={() => setShowPreviewFeaturesModal(true)}>
+                    <FormattedMessage id="PreviewFeatures" defaultMessage="Preview Features" />
+                    <Span
+                      fontSize="12px"
+                      ml={1}
+                      style={{ borderRadius: 20 }}
+                      py="2px"
+                      px="6px"
+                      display="inline-block"
+                      bg="primary.200"
+                      color="black.800"
+                    >
+                      <FormattedMessage defaultMessage="New!" />
+                    </Span>
+                  </StyledMenuEntry>
+                )}
+                <hr />
+                <StyledMenuEntry href={`/create`}>
+                  <FormattedMessage defaultMessage="New Collective" /> <Plus size={14} />
+                </StyledMenuEntry>
+                <StyledMenuEntry href={`/organizations/new`}>
+                  <FormattedMessage defaultMessage="New Organization" /> <Plus size={14} />
+                </StyledMenuEntry>
+                <hr />
+                <StyledMenuEntry href={`/home`} openInNewTab>
+                  <FormattedMessage defaultMessage="Open Collective Homepage" /> <ExternalLink size={14} />
+                </StyledMenuEntry>
+                <StyledMenuEntry href={`/help`} openInNewTab>
+                  <FormattedMessage defaultMessage="Help & Support" /> <ExternalLink size={14} />
+                </StyledMenuEntry>
+                <StyledMenuEntry href={`https://docs.opencollective.com`} openInNewTab>
+                  <FormattedMessage defaultMessage="Documentation" /> <ExternalLink size={14} />
+                </StyledMenuEntry>
+                <hr />
+                <StyledMenuEntry as="button" onClick={() => logout()}>
+                  <FormattedMessage id="menu.logout" defaultMessage="Log out" /> <LogOut size={14} />
+                </StyledMenuEntry>
+              </Flex>
+            </StyledDropdownContent>
+          </StyledDropdownContainer>
+        )}
+      </Dropdown>
+      {showPreviewFeaturesModal && <PreviewFeaturesModal onClose={() => setShowPreviewFeaturesModal(false)} />}
+    </React.Fragment>
+  );
+};
+export default withNewsAndUpdates(ProfileMenuDropdown);
```

Comment:
Is this even being used?
If so, I would rather for us to use `useNewsAndUpdates` hook instead of the HOC.

```suggestion
export default ProfileMenuDropdown;
```

Reply:
It's used on the mobile version:
<img width="390" alt="image" src="https://github.com/opencollective/opencollective-frontend/assets/1552194/2025fd67-c9e9-4b2a-83c8-04fb2729f64e">
Although arguably difficult to use since you have to hit the little fire icon to trigger the modal.

Anyway, good point about the hook, fixing that!